TAnas0/Apollo-Server-Boilerplate
# Update to support Node 22

### Description
This PR update the project and its dependencies to work with version 22.7.0 of Node, and the corresponding version 10.8.2 of NPM.

An update from Webpack 3 to Webpack 4 was necessary.
Update of ESLint was necessary, and required minor stylistic changes.

### Related Issues
- Closes #50 
- Relates to #48 


### Testing Instructions
Make sure to use version 22 of Node (using nvm), then install dependencies from scratch and test running the development server:

```bash
rm -rf node_modules
npm cache clean --force
npm install
npm run dev
```
